rosa/cli.py

```diff
@@ -152,99 +141,50 @@ def get_available_flags(command):
-                        )
-                        commands_dict[top_command][command][_command][
-                            "auto_mode"
-                        ] = check_flag_in_flags(
-                            command_list=[rosa_cmd, top_command, _command],
-                            flag_str=auto_mode_str,
-                        )
-                        commands_dict[top_command][command][_command][
-                            "region"
-                        ] = check_flag_in_flags(
-                            command_list=[rosa_cmd, top_command, _command],
-                            flag_str=region_str,
-                        )
+                        key_path = [top_command, command, _command]
+                        commands_dict[key_path] = {}
+                        _fill_path(key_path)
                 else:
                     # If sub command doesn't have sub command
-                    commands_dict[top_command][command][
-                        "json_output"
-                    ] = check_flag_in_flags(
-                        command_list=[rosa_cmd, top_command, command],
-                        flag_str=output_flag_str,
-                    )
-                    commands_dict[top_command][command][
-                        "auto_answer_yes"
-                    ] = check_flag_in_flags(
-                        command_list=[rosa_cmd, top_command, command],
-                        flag_str=auto_answer_yes_str,
-                    )
-                    commands_dict[top_command][command][
-                        "auto_mode"
-                    ] = check_flag_in_flags(
-                        command_list=[rosa_cmd, top_command, command],
-                        flag_str=auto_mode_str,
-                    )
-                    commands_dict[top_command][command]["region"] = check_flag_in_flags(
-                        command_list=[rosa_cmd, top_command, command],
-                        flag_str=region_str,
-                    )
-
+                    _fill_path(key_path)
         else:
             # If top command doesn't have sub command
-            commands_dict[top_command]["json_output"] = check_flag_in_flags(
-                command_list=[rosa_cmd, top_command],
-                flag_str=output_flag_str,
-            )
-            commands_dict[top_command]["auto_answer_yes"] = check_flag_in_flags(
-                command_list=[rosa_cmd, top_command],
-                flag_str=auto_answer_yes_str,
-            )
-            commands_dict[top_command]["auto_mode"] = check_flag_in_flags(
-                command_list=[rosa_cmd, top_command],
-                flag_str=auto_mode_str,
-            )
-            commands_dict[top_command]["region"] = check_flag_in_flags(
-                command_list=[rosa_cmd, top_command],
-                flag_str=region_str,
-            )
+            _fill_path(key_path)
 
-    return commands_dict
+    return dict(commands_dict)
```

Comment:
`build_command` converts the dict back to benedict; why is this needed?

Reply:
For the case that `parse_help` is called separately as standalone function, not as part of `execute`.

Reply:
benedict has extended functionality but also provides anything dict provides - why not return the object as is?